package.json

```diff
@@ -55,13 +69,16 @@
     "react-native": ">=0.50.0",
     "react-native-svg": ">=12.1.0"
   },
-  "release": {
```

Comment:
@JoseLion I am happy to see you extracted out `semantic-release` configuration to a separate file, IMHO keeping all the configuration in the same file is less readable

Reply:
I agree! I noticed that when I was looking for the `semantic-release` config and I couldn't find it. I thought this OR would be a great opportunity to change that 🙂